includes/sensei-functions.php

```diff
@@ -29,8 +29,10 @@ function is_sensei() {
 	) {
 		$is_sensei = true;
 	} elseif ( is_object( $post ) && ! is_wp_error( $post ) ) {
-		$course_page_id     = intval( Sensei()->settings->settings['course_page'] );
-		$my_courses_page_id = intval( Sensei()->settings->settings['my_course_page'] );
+		$course_page_id           = intval( Sensei()->settings->settings['course_page'] );
+		$my_courses_page_id       = intval( Sensei()->settings->settings['my_course_page'] );
+		$course_completed_page_id = intval( Sensei()->settings->settings['course_completed_page'] );
```

Comment:
Just an unused variable here.

Reply:
Fixed in https://github.com/Automattic/sensei/pull/4273/commits/fa3f34d75da02de28e9b119cd542424bcc45f198.